### PR TITLE
Tweaks while reading through notebooks

### PR DESCRIPTION
**Pull request by drphilmarshall**
Hopefully easy to merge, @abmantz !

### CODE REVIEW

File: lessons/basic_mc.ipynb [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "In other words, given a list of samples of $\\theta$ from $P(\\theta)$,\n",
     "* the marginalized 1D posterior for $\\theta_0$ is estimated by making a histogram of $\\theta_0$ samples\n",
     "* the marginalized 2D posterior for $\\theta_0,\\theta_1$ is estimated from a 2D histogram of $\\theta_0,\\theta_1$ samples\n",
+    "* the posterior mean of $\\theta_0$ is estimated by a simple average over the $\\theta_0$ samples\n",
```
Comment (abmantz) [inline lessons/basic_mc.ipynb]: I don't really see the value in adding this. Going from a posterior to means and other estimates is covered earlier, and posterior means are not something I want to promote particularly.
```suggestion
```

File: lessons/bayes_theorem.ipynb [hunk 2]
```diff
@@ -734,7 +734,6 @@
    "cell_type": "code",
```
Comment (abmantz) [inline lessons/bayes_theorem.ipynb:734]: It's really annoying that the PR is cluttered with files where only the metadata has changed due to saving the notebook.

File: lessons/probability.ipynb [hunk 3]
```diff
@@ -75,7 +75,7 @@
     "\n",
     "Let $\\Omega$ be the set of states available to a system of fixed energy, e.g. a box full of gas particles.\n",
     "\n",
-    "With one additional (physics) assumption, that it's equally probable for the system to occupy any state in $\\Omega$, this is the microcanonical ensemble in statistical mechanics."
+    "With one additional (physics) assumption, that it's equally probable for the system to occupy any state in $\\Omega$. This is the microcanonical ensemble in statistical mechanics."
```
Comment (abmantz) [inline lessons/probability.ipynb]: Modification is grammatically wrong.
```suggestion
    "With one additional (physics) assumption, that it's equally probable for the system to occupy any state in $\\Omega$, this is the microcanonical ensemble in statistical mechanics."
```

File: lessons/probability.ipynb [hunk 4]
```diff
@@ -87,7 +87,7 @@
    },
    "source": [
     "### Discrete vs. continuous sample spaces\n",
-    "Very often the type of event we're interested in lives in a continuous sample space, e.g. the event $h=0.7$.\n",
+    "Very often the type of event we're interested in lives in a continuous sample space. e.g. The universe is such that $h=0.7$.\n",
```
Comment (abmantz) [inline lessons/probability.ipynb]:
```suggestion
    "Very often the type of event we're interested in lives in a continuous sample space. E.g., the Hubble parameter is $h=0.7$.\n",
```

File: lessons/probability.ipynb [hunk 5]
```diff
@@ -221,6 +221,7 @@
    },
    "source": [
     "$P(Y|X)$ is a (normalized) slice through $P(X,Y)$ rather than an integral.\n",
+    "> _In the lefthand panel, imagine a vertical cut at $x=2$, to give us $P(y|x=2)$ in the righthand panel._\n",
```
Comment (abmantz) [inline lessons/probability.ipynb]: I'd prefer to issue this as a figure improvement (show the cut at x=2).
```suggestion
```

**Review by abmantz — CHANGES_REQUESTED** [review]

File: lessons/basic_mc.ipynb (hunk 1 above)
Comment (drphilmarshall) [reply lessons/basic_mc.ipynb]: Fair enough. I just know that when I was learning this stuff, the posterior mean was an easier integral to relate to than the histogram bin values.

File: lessons/bayes_theorem.ipynb (hunk 2 above)
Comment (drphilmarshall) [reply lessons/bayes_theorem.ipynb:734]: I feel your pain.

File: lessons/probability.ipynb (hunk 5 above)
Comment (drphilmarshall) [reply lessons/probability.ipynb]: Fine with me.

File: lessons/basic_mc.ipynb (hunk 1 above)
Comment (abmantz) [reply lessons/basic_mc.ipynb]: I think I will add back a comment about computing means and percentiles from samples later.

**Pull request merged**